src/pages/workspace/companyCards/addNew/CardNameStep.tsx

```diff
@@ -0,0 +1,87 @@
+import React, {useCallback} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import FormProvider from '@components/Form/FormProvider';
+import InputWrapper from '@components/Form/InputWrapper';
+import type {FormInputErrors, FormOnyxValues} from '@components/Form/types';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Text from '@components/Text';
+import TextInput from '@components/TextInput';
+import useAutoFocusInput from '@hooks/useAutoFocusInput';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as ValidationUtils from '@libs/ValidationUtils';
+import * as CompanyCards from '@userActions/CompanyCards';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import INPUT_IDS from '@src/types/form/AddNewCardFeedForm';
+
+function CardNameStep() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const {inputCallbackRef} = useAutoFocusInput();
+    const [addNewCard] = useOnyx(ONYXKEYS.ADD_NEW_COMPANY_CARD);
+
+    const validate = useCallback(
+        (values: FormOnyxValues<typeof ONYXKEYS.FORMS.ADD_NEW_CARD_FEED_FORM>): FormInputErrors<typeof ONYXKEYS.FORMS.ADD_NEW_CARD_FEED_FORM> => {
+            const errors = ValidationUtils.getFieldRequiredErrors(values, [INPUT_IDS.CARD_TITLE]);
+            if (!values.cardTitle) {
+                errors.cardTitle = translate('common.error.fieldRequired');
+            }
+            return errors;
+        },
+        [translate],
+    );
+
+    const submit = useCallback((values: FormOnyxValues<typeof ONYXKEYS.FORMS.ADD_NEW_CARD_FEED_FORM>) => {
+        CompanyCards.setAddNewCompanyCardStepAndData({
+            step: CONST.COMPANY_CARDS.STEP.CARD_DETAILS,
+            data: {
+                cardTitle: values.cardTitle,
+            },
+            isEditing: false,
+        });
+    }, []);
+
+    const handleBackButtonPress = useCallback(() => {
```

Comment:
Why do we need `useCallback` here (and in the above calls in this file) but not in the other files?